smallclib/Makefile

```diff
@@ -0,0 +1,140 @@
+P	= mips-mti-elf-
+CC	= $(P)gcc -mips32r2 -EL -nostdlib
+AR  = $(P)ar
+
+# -Werror disabled for building a third-party external lib
+CFLAGS =  -Wall -DPIC32MZ
+# No optimisations, emit debugging info.
+CFLAGS += -O0 -g
```

Comment:
Why not just `-Og` i.e. optimizations that do not interfere with generating debugging information ?